stdlib/ballerina-builtin/src/main/ballerina/os/natives.bal

```diff
@@ -37,3 +37,11 @@ public native function getVersion () returns (string);
 @Description {value:"Returns the architecture of the operating system."}
 @Return { value:"OS architecture if the OS can be identified, an empty string otherwise"}
 public native function getArchitecture () returns (string);
+
+@Description { value:"Execute a given CMD command or a script"}
+@Param {value:"command: Command to be executed"}
+@Param {value:"workingDir: Execution workspace, the command will be executed from here"}
+@Param {value:"timeoutValue: Timeout value for the process to exit, if this is exceeded the process is killed"}
+@Return { value:" Returns a error if the execution is not successfull or user timeout exceeds"}
+public native function runCommand(@sensitive string command, string workingDir = "", int timeoutValue = -1)
```

Comment:
Is there a need to allow an end-user input to influence the 'workingDir'? IMO, workingDir should also be "@ sensitive". Otherwise, we will be opening up a situation where an end-user is allowed to change the workingDir of a command getting executed. 

(Example: "rm -rf" on a wrong workingDir)

Same logic applies to "timeoutValue". End-user being able to control the timeout can lead to DOS attack possibility.

Reply:
@ayomawdb actually the full control over the command execution is on the end-user. And the workingDir and timeOut is bound to the current execution command. The only way to restrict the user is by having proper set of privileges to the process initiating user.  But +1 for making the working directory sensitive. But in my opinion timeout shouldn't have to be sensitive, If the command execution ends the process gets terminated irrespective of the timeout provided.